.github/workflows/zwave-js-bot.yml

```diff
@@ -0,0 +1,120 @@
+name: "Z-Wave Bot: React to comments"
+
+on:
+  issue_comment:
+    types: [created] # edited, deleted
+
+jobs:
+  # #########################################################################
+  # Fix lint errors when an authorized person posts "@zwave-js-bot fix lint"
+  fix-lint:
+    if: |
+      contains(github.event.issue.html_url, '/pull/') &&
+      contains(github.event.comment.body, '@zwave-js-bot fix lint') &&
+      (github.event.comment.user.login != 'zwave-js-bot')
+
+    runs-on: [ubuntu-latest]
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: Checkout master branch
+        uses: actions/checkout@v2
+
+      - name: Check user's permissions to do this
+        id: check
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{secrets.BOT_TOKEN}}
+          result-encoding: string
+          script: |
+            const bot = require(`${process.env.GITHUB_WORKSPACE}/.github/bot-scripts/index.js`);
+            return await bot.fixLintCheck({github, context});
```

Comment:
You'll need to copy these files from my repo

Reply:
What if we create a `.github` repo in the org with them? Should it work?

Reply:
https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-default-community-health-file

Reply:
No, that runs the cloned scripts defined in your repo. Could maybe make it an action that we share between repos.

Reply:
Nope I don't think it will work, but maybe could be useful for other purposes :)

Reply:
That doesn't mention scripts, but could come in handy.